manual/qmcpack_manual.sty

```diff
@@ -0,0 +1,34 @@
+\usepackage{ifpdf}
+\ifpdf
+%this supresses the moaning about unicode characters
+\usepackage[T1]{fontenc}
+\usepackage[utf8]{inputenc}
+\makeatletter
+\def\UTFviii@defined#1{%
+  \ifx#1\relax
+      ?%
+  \else\expandafter
+    #1%
+  \fi
+}
+\makeatother
+\else
+\usepackage{fontspec}
+%% We need to pick a font the system has..
+%% default on linux and osx lacks unicode box characters
+%% The empty bracket below is false condition of the Courier New Check
+\@ifpackageloaded{fontspec}%
+                 {%
+\setmainfont{XCharter Roman}%
+\@ifpackagelater{fontspec}{2016/01/30}% Guess at when IfFontExistsTF was added.
```

Comment:
The date on the fontspec package in Ubuntu 16.04 is 2016/02/01 (v2.5a), and it does not have IfFontExistsTF.
 Changing the test date to 2016/02/02 works.

(The commit where IfFontExistsTF was added occurs on Dec 27, 2016)
https://github.com/wspr/fontspec/commit/6969d3ad086391be4655779104ef83fb0eee23fb
It appears first in tag 2.5c and the v2.5c release date on github is 01/20/2017